lighthouse-cli/performance-experiment/report/scripts/perf-x.js

```diff
@@ -83,15 +84,6 @@ class ConfigPanel {
         }
       });
     });
-
-    // get and recover blocked URL patterns of current run
-    fetch(`/flags?id=${this._reportId}`).then(response => {
```

Comment:
you could also use a polyfill for this: 
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-viewer/package.json#L34